streaming/streaming.go

```diff
@@ -445,16 +445,22 @@ func LaunchDataflowJob(ctx context.Context, targetProfile profiles.TargetProfile
 	} else {
 		dataflowHostProjectId = dataflowCfg.HostProjectId
 	}
+
 	dataflowSubnetwork := ""
+
+	// if custom network is not selected, use public IP. Typical internal testing flow
+	workerIpAddressConfig := dataflowpb.WorkerIPAddressConfiguration_WORKER_IP_PUBLIC
```

Comment:
should this be PUBLIC or UNSPECIFIED

Reply:
PUBLIC. This gives the default behavior that exists when the public IP is not explicitly disabled.